packages/plugin-constraints/sources/Constraints.ts

```diff
@@ -42,14 +95,20 @@ class Session {
   }
 
   public async *makeQuery(query: string) {
-    this.session.query(query);
+    const parsed = this.session.query(query) as any;
```

Comment:
We control the types of tau-prolog ourselves, so you could modify the type to have this function return `true|pl.type.Term<any, any>` instead of casting to `any`